<?xml version="1.1" ?>

<robot name="mrm" xmlns:xacro="http://www.ros.org/wiki/xacro">

  <material name="blue">
    <color rgba="0 0 2.2 1"/>
  </material>

  <material name="white">
    <color rgba="1 1 1 1"/>
  </material>

  <link name="base_link">
      <visual>
          <origin rpy="0 1.575 0" xyz="0 0 0.0"/>
          <geometry>
              <cylinder radius="0.35" length="0.0"/>
          </geometry>
          <material name="black"/>
      </visual>
  </link>
  <joint name="joint0" type="revolute">
      <axis xyz="0 0 1" />
      <limit effort="1000.0" lower="-3.14" upper="3.14" velocity="0.5" />
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <parent link="base_link"/>
      <child link="link2"/>
  </joint>
  
  <link name="link2">
      <visual>
          <origin rpy="0 1.575 0" xyz="1.1 0 0"/>
          <geometry>
              <cylinder radius="0.1" length="2.2"/>
          </geometry>
          <material name="white"/>
      </visual>
      <collision>
        <geometry>
          <cylinder length="2.2" radius="0.1"/>
        </geometry>
      </collision>
      <inertial>
        <mass value="0.6"/>
        <inertia ixx="0.001" ixy="0.0" ixz="0.0" iyy="0.001" iyz="0.0" izz="0.001"/>
      </inertial>
  </link>

  <joint name="joint1" type="revolute">
      <axis xyz="0 0 1" />
      <limit effort="1000.0" lower="-3.14" upper="3.14" velocity="0.5" />
      <origin rpy="0 0 0" xyz="2.2 0 0"/>
      <parent link="link2"/>
      <child link="link3"/>
  </joint>
  
  <link name="link3">
      <visual>
          <origin rpy="0 1.575 0" xyz="1.1 0 0"/>
          <geometry>
              <cylinder radius="0.1" length="2.2"/>
          </geometry>
          <material name="blue"/>
      </visual>
      <collision>
        <geometry>
          <cylinder length="2.2" radius="0.1"/>
        </geometry>
      </collision>
      <inertial>
        <mass value="0.6"/>
        <inertia ixx="0.001" ixy="0.0" ixz="0.0" iyy="0.001" iyz="0.0" izz="0.001"/>
      </inertial>
  </link>
</robot>